### ISSUE TITLE
Can't compile MangoHud in Ubuntu 24.04.1 LTS

### ISSUE THREAD

**Issue by Slo2018** [flightlessmango/MangoHud#1490]
I got error that something missing (X 11) when i compiling MangoHud. It worked in Linux Mint 22 but not in Ubuntu.

**Comment by 17314642** [comment]
Send your terminal output when you try to build mangohud.

**Comment by ericzlmd** [comment]
I have the following issue as well:

`
Program python3 (mako) found: YES (/usr/bin/python3) modules: mako
Checking if "GCC atomic builtins" compiles: YES 
Checking if "Supports timespec_get" compiles: YES 
Checking for function "bswap32" : YES 
Checking for function "bswap64" : YES 
Checking for function "clz" : YES 
Checking for function "clzll" : YES 
Checking for function "ctz" : YES 
Checking for function "expect" : YES 
Checking for function "ffs" : YES 
Checking for function "ffsll" : YES 
Checking for function "popcount" : YES 
Checking for function "popcountll" : YES 
Checking for function "unreachable" : YES 
Found pkg-config: YES (/usr/bin/pkg-config) 2.3.0
Found CMake: /usr/bin/cmake (3.30.5)
Run-time dependency x11 found: NO (tried pkgconfig and cmake)

meson.build:88:12: ERROR: Dependency "x11" not found, tried pkgconfig and cmake
`
CPU: 9800x3D
OS: Fedora 41
GPU: RTX 3080ti

**Comment by 17314642** [comment]
@ericzlmd does installing `libX11-devel` solve the issue?

**Comment by ericzlmd** [comment]
> @ericzlmd does installing `libX11-devel` solve the issue?

Yes. The resolved it! How about for wayland-client, I have libwayland-client.x86_64 installed.

**Comment by 17314642** [comment]
@ericzlmd just do `dnf builddep mangohud`

**Comment by ericzlmd** [comment]
@17314642 major TIL. Many many thanks!
